packages/@aws-cdk/aws-ec2-alpha/test/integ.route-v2.ts

```diff
@@ -115,6 +115,20 @@ const dynamoEndpoint = new GatewayVpcEndpoint(stacks.dynamodb, 'testDynamoEndpoi
 });
 routeTables.dynamodb.addRoute('dynamoRoute', '0.0.0.0/0', { endpoint: dynamoEndpoint });
 
+vpcs.vpcpc2 = new vpc_v2.VpcV2(stacks.vpcpc, 'secondVPC', {
+  primaryAddressBlock: vpc_v2.IpAddresses.ipv4('10.1.0.0/16'),
+  secondaryAddressBlocks: [vpc_v2.IpAddresses.ipv4('10.2.0.0/16', { cidrBlockName: 'Temp Block' })],
+  region: 'us-west-2',
+  ownerAccountId: '012345678910',
+});
+vpcs.vpcpc2.createAcceptorVpcRole('987654321098');
```

Comment:
in place of hardcoded account values here can we add a variable and some instructions to run it like we have in 
https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk-testing/framework-integ/test/aws-route53/test/integ.cross-account-zone-delegation.ts , also i'll prefer to move it into a separate test so it doesn't cause any issue with the existing test for route, but in case its too much let's just the add the instructions and variable at the top.

Reply:
agreed, done! [9ff9da6](https://github.com/aws/aws-cdk/pull/31645/commits/9ff9da66cd2b728ebe82e50e38ec8e2c188038d1)